presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -98,6 +100,24 @@ public static Block createStringsBlock(Iterable<String> values)
         return builder.build();
     }
 
+    public static Block createBinaryLiteralsBlock(String ...values)
+    {
```

Comment:
Rename this `createSlicesBlock`. Split this into 2 functions like `createStringsBlock`.